### DISCUSSION TITLE
Delete of old containers

### DISCUSSION THREAD

**Discussion by Goten87** [louislam/dockge#523] [General]
if i update a container will it delete the old img files or will it keep it 
if it will keep it how do i delete them to save space

**Comment by thierolm** [comment]
U can execute docker cli command `docker image prune -a -f` to remove unused images.

![image](https://github.com/louislam/dockge/assets/77847348/1cdd36f6-4c02-4e38-a102-a8421745e04b)

@louislam it would be cool to have an option to automatically execute the `prune` command after updating a container. :-)
